### DISCUSSION TITLE
Log file Implementation

### DISCUSSION THREAD

**Discussion by djp3k05** [madMAx43v3r/chia-plotter#346] [Ideas]
Hi,

In order to be able to use any Plot Manager, we need a log file. Can you implement that? The standard log file output would be ideal, in order not to mess to much with the plot managers. Also, it will be easier if you CLI parameters would be the same a official chia commands.

Regards and congrats for you amazing work.

**Comment by djp3k05** [comment]
just found this https://github.com/swar/Swar-Chia-Plot-Manager/pull/1198

**Reply by grayfallstown** [reply]
For linux use `(your chia_plot command line)  2>&1 | tee /home/$(whoami)/.chia/mainnet/plotter/chia-plotter-$(uuid).log;` or `(your chia_plot command line) 2>&1 | % ToString | Tee-Object -FilePath "C:\Users$env:UserName.chia\mainnet\plotter$([GUID]::NewGUID().ToString('D')).log"` for windows.

**Reply by djp3k05** [reply]
thanks, it worked (linux)
